src/Whip_WPDismissOption.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Represents the WordPress option for saving the dismissed messages.
+ */
+class Whip_WPDismissOption implements Whip_DismissStorage {
+
+	/**
+	 * Saves the value to the options.
+	 *
+	 * @param string $dismissedVersion The value to save.
+	 */
+	public function set( $dismissedVersion ) {
+		update_option( 'whip_dismissed', $dismissedVersion );
```

Comment:
The option name is not very descriptive of the value it will contain.
Please consider renaming this to something like: `whip_dismissed_for_wp_version`